su-template/default/module/html/method_summary.erb

```diff
@@ -0,0 +1,15 @@
+<% if method_listing.size > 0 %>
+  <% groups(method_listing) do |list, name| %>
+    <a name="<%= name.gsub(' ', '_').downcase %>"></a>
+    <h2>
+      <%= name %>
```

Comment:
It appear to me that the anchor, the `name` attribute, is better fit here, wrapping the title. The semantics of the document is better like that.